src/components/editor/secretsEditor.js

```diff
@@ -8,26 +8,32 @@ export default class A0SecretsEditor extends React.Component {
     constructor(props) {
         super(props);
 
-        const secrets = Object.keys(props.secrets)
+        this._loadSecrets();
+
+        this.keyRefs = [];
```

Comment:
Can you dump this and the next line as well? They are no longer relevant it seems.